src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/BasicHtmlWebResponseObject.Common.cs

```diff
@@ -18,12 +18,24 @@ public class BasicHtmlWebResponseObject : WebResponseObject
     {
         #region Private Fields
 
-        private static Regex s_attribNameValueRegex;
-        private static Regex s_attribsRegex;
-        private static Regex s_imageRegex;
-        private static Regex s_inputFieldRegex;
-        private static Regex s_linkRegex;
-        private static Regex s_tagRegex;
+        private static readonly Regex s_attribsRegex = CreateAttribsRegex();
+        private static readonly Regex s_attribNameValueRegex = CreateAttribNameValueRegex();
+        private static readonly Regex s_imageRegex = CreateImageRegex();
+        private static readonly Regex s_inputFieldRegex = CreateInputFieldRegex();
+        private static readonly Regex s_linkRegex = CreateLinkRegex();
+        private static readonly Regex s_tagRegex = CreateTagRegex();
```

Comment:
I guess they are always initialized but we want they are initialized only at first use time. You could check this in debugger.
My proposal was tp put the code in new static helper class. It will be initialized only we first use it.

Reply:
Does the last commit, solve this correctly?